README.md

```diff
@@ -203,6 +237,7 @@ GPU, much faster than previous methods, while the generated instances can be dir
 - [3D Gaussian Splatting](https://github.com/WangFeng18/3d-gaussian-splatting)
 - [fast: C++/CUDA](https://github.com/MrNeRF/gaussian-splatting-cuda)
 - [nerfstudio: python/CUDA](https://github.com/nerfstudio-project/gsplat)
+- [gaussian-splatting-Windows](https://github.com/jonstephens85/gaussian-splatting-Windows)
```

Comment:
I think this repo is a bit redundant here. I rather like it removed as it is also linked in the youtube description of the video you added below. It does not add much useful information and does not change the code in comparison to the original implementation.

Reply:
True enough. And happily the original source README.md does include a link to this source, so you can delete. I found it (esp. the tutorial and the YouTube video notes) quite helpful in getting everything actually working. But, it's just a click away, for someone looking at the original.